server/controllers/ssotRetrievalController.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const ssotModels = require('../models/singleSourceOfTruthModel.js');
+
+
+// GET /ssot/id
```

Comment:
Shouldn't we use it like /ssot?id=12345 ?

Reply:
in ssot.js you also implemented it differently. There it is ssot/get/:id. What is the right version?
I would prefer something like /ssot/get?id=12345 so kind of a mixture of both :)

Reply:
It would be ark-automate.com/ssot/get/42069 where 42069 is the id of that user. Specifying a parameter is not necessary imo

Reply:
If that works I am fine with it but in line 10 you are accessing the parameter id if I am not mistaken so we would need a parameter id if we do it like that, right?

Reply:
this id is the automatically assigned id by mongoose. It automatically generates one for each database entry

Reply:
I get it now I got confused with query parameters and URI parameters :D